embedding/cpp/include/elm_flutter_view.h

```diff
@@ -0,0 +1,60 @@
+// Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef FLUTTER_TIZEN_EMBEDDING_CPP_INCLUDE_FLUTTER_VIEW_H_
+#define FLUTTER_TIZEN_EMBEDDING_CPP_INCLUDE_FLUTTER_VIEW_H_
+
+#include <flutter/plugin_registry.h>
+#include <flutter_tizen.h>
+
+#include <string>
+#include <vector>
+
+// The app base class which creates and manages the Flutter engine instance.
+class ElmFlutterView : public flutter::PluginRegistry {
+ public:
+  ElmFlutterView(void *elm_parent) : elm_parent_(elm_parent) {}
+  virtual ~ElmFlutterView() {}
+
+  virtual bool OnCreate();
+
+  FlutterDesktopPluginRegistrarRef GetRegistrarForPlugin(
+      const std::string &plugin_name) override;
+
+  bool IsRunning() { return engine_ != nullptr; }
+
+  void *GetEvasImageHandle() { return evas_image_; };
+
+  // The width of the view, or the maximum width if the value is zero.
+  int32_t window_width_ = 0;
```

Comment:
`window_width_` and `window_height_` can be set again after starting the View engine for now.
How about providing getter methods instead?

Reply:
Thank you for review.
I added Getter and Resize methods.
This is resize test image (https://github.com/JSUYA/jsuya.github.io/blob/master/ElmFlutterView_test.gif)

Reply:
@WonyoungChoi Anything more to discuss?